src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,6 +1,6 @@
 package mate.academy.service;
 
-public class AuthenticationService {
+public class AuthenticationService extends UserService {
```

Comment:
It would be better to use composition over inheritance here, because we can not say that `AuthenticationService` IS A `UserService`, so let's make `UserService` a field of an `AuthenticationService`
more about it [here](https://www.digitalocean.com/community/tutorials/composition-vs-inheritance)